src/librbd/librbd.cc

```diff
@@ -5725,6 +5730,7 @@ extern "C" int rbd_group_image_list(rados_ioctx_t group_p,
   tracepoint(librbd, group_image_list_enter,
              group_ioctx.get_pool_name().c_str(),
 	     group_ioctx.get_id(), group_name);
+  memset(images, 0, sizeof(rbd_group_image_info_t) * *image_size);
```

Comment:
Nit: why not ```sizeof(*images)``` here like you used elsewhere?

Reply:
@dillaman  It seems that it can't work as expected.   e.g. :
```
-  memset(images, 0, sizeof(rbd_group_image_info_t) * *image_size);
+  memset(images, 0, sizeof(*images));

   if (group_image_info_size != sizeof(rbd_group_image_info_t)) {
     *image_size = 0;
@@ -5747,6 +5747,7 @@ extern "C" int rbd_group_image_list(rados_ioctx_t group_p,
     return 0;
   }

+  r = -EINVAL;		// just for test
   if (r < 0) {

>>> import rbd,rados;client=rados.Rados(conffile='');client.connect();ioctx=client.open_ioctx('rbd');group1=rbd.Group(ioctx, 'group1')
>>> list(group1.list_images())
*** Error in `python': munmap_chunk(): invalid pointer: 0x0000000002063cdc ***
======= Backtrace: =========
/lib64/libc.so.6(+0x7f5e4)[0x7f43976a75e4]
/home/swb/ceph/ceph1/build/lib/librbd.so.1(rbd_group_image_list_cleanup+0x5a)[0x7f438feb2c8a]
```

Reply:
```
 - memset(images, 0, sizeof(rbd_group_image_info_t) * *image_size);
+ memset(images, 0, sizeof(*images));
```
Did you mean to do ```memset(images, 0, sizeof(*images) * image_size)``` so that the whole array is cleared?

Reply:
@dillaman  Yeah, I think it (`memset(images, 0, sizeof(*images) * *image_size)`) does.

> why not sizeof(*images) here like you used elsewhere?

BTW: sorry, I misunderstood your point of  this view at fisrt sight. : )